samples/04.1-file-transfer-listener/file-transfer-listener-integration-tests/src/test/java/org/eclipse/edc/sample/extension/FileTransferListenerSampleTest.java

```diff
@@ -76,11 +76,6 @@ void runSampleSteps() throws Exception {
         assertFileContent(MARKER_FILE, MARKER_FILE_CONTENT);
     }
 
-    @AfterEach
```

Comment:
Please move `Before` and `After` methods well visible on the top of the `Tests` ones, at least above the test private methods